### DISCUSSION TITLE
Inav 4.0.0 - Unexpected motor stop and drone crash - defective battery or something else  ?

### DISCUSSION THREAD

**Discussion by alccrl** [iNavFlight/inav#7725] [Q&A]
Hi,
I was in doubt about opening an issue on inav 4.0.0 because I cannot reproduce the problem in short times, but while I was piloting in ACRO mode in a very soft way and no wind, quad not so far from me, so in complete control of what it was going on, suddenly the quad pointed to ground and crashed. !!!
I reacted putting quad into POS_HOLD, but it was too late, quad was already in the mud (luckily - not crashed on the rocks).
I attach here to this post
- taranis opentx log
- blackbox log
- short footage from goggles fpv recording
- screenshot  of blackbox explorer with an arrow indicating the crash point

Equipment used: 
AutoBuilt Fpv Long Range QAV-250 frame with Matek F722-STD FC and Taranis X9-Lite RC link (868-900 Mhz)), 6 inch propellers. Inav 4.0.0 official release. 5.8 Ghz video link (vtx).
![Screenshot_BlakBoxExplorer](https://user-images.githubusercontent.com/78967144/148601869-25e3b6e0-c024-4d6d-b394-9fd476a6ae1b.jpg)
[THE_BEAST-2022-01-07.csv](https://github.com/iNavFlight/inav/files/7831123/THE_BEAST-2022-01-07.csv)
[LOG00101.TXT](https://github.com/iNavFlight/inav/files/7831124/LOG00101.TXT)

https://user-images.githubusercontent.com/78967144/148597096-dbcd677f-5264-45c4-af25-32eed8e07063.mp4

YOU CAN SEE something like a HOLE in the bottom state bar, at about time 00:30. it seems that sw stopped control of motors ...

Any help about this ?
Could it be a defective battery casuing the problem ?

Any help is appreciated !!!
Thanks in advance ...

Alfredo

**Comment by alccrl** [comment]
also add Inav 4.0.0 configuration file.

A
[INAV_cli_20220107_214650.txt](https://github.com/iNavFlight/inav/files/7831277/INAV_cli_20220107_214650.txt)
lfredo

**Reply by alccrl** [reply]
Issue #7726 has been closed because an explanation exists for this behaviour. If behaviour is as expected this does not mean that behaviour is good, some problem about requirements might arise that suggest a different behaviour or solution.
So, I consider issue #7726 still open.

**Comment by Odin263** [comment]
Maybe this would be an option to avoid such problems?:
![image](https://user-images.githubusercontent.com/90968252/153580038-dc526546-7af0-45ae-b170-c1bdf0d72bb3.png)
So if the speed is greater than a certain value or throttle is greater than threshold, airmode should be active. So the airmode should activate even if the quad doesn't move in the air and then starts to fall down.

**Reply by MrD-RC** [reply]
Airmode keeps stabilisation active at LOW throttle. When you raise the throttle, airmode disengages. That’s what its supposed to do. If the throttle is not LOW, the stabilisation works fine anyway.

The problem was that the throttle didn’t go high enough to activate airmode in the first place. Paweł has since lowered to the activation threshold. So this problem will not happen.

**Reply by alccrl** [reply]
I anticipated in my configuration a lower threshold (1150) and my quad luckily is still alive ... the change seems to work.
I think I will move to Inav 4.1 as soon as it is ready and I will check thresholds again.

Alfredo

**Reply by alccrl** [reply]
Lowering air mode trigger threshold is the simplest workaround. Inav 4.1 implements this.
So, I guess  this discussion is closed.
